docs/API.Stream.ConsumeNextChunk.md

```diff
@@ -41,26 +43,172 @@ position | Defines the position of the chunk in the stream |  Position ID return
 
 ### Call ###
 
-    curl -vX GET <cm-well-host>/example.org/Individuals/_consume?position=eQA5eJwzNDE1sDQ3NDI0tjQ2rjGoMTE0MDI1NjS0BIEa_dSKxNyCnFS9_KJ0fc-8lMyyzJTSxJzimpoa&format=json
+    curl -vX GET <cm-well-host>/example/Geography/_consume?position=eQA5eJwzNDE1sDQ3NDI0tjQ2rjGoMTE0MDI1NjS0BIEa_dSKxNyCnFS9_KJ0fc-8lMyyzJTSxJzimpoa&format=json
```

Comment:
- _consume is a root endpoint: `<cm-well-host>/_consume` (drop the `/example/Geography` part)
- position is wrong:
```
sbt ws/console
...
scala> import cmwell.util.string._
import cmwell.util.string._

scala> val s = "eQA5eJwzNDE1sDQ3NDI0tjQ2rjGoMTE0MDI1NjS0BIEa_dSKxNyCnFS9_KJ0fc-8lMyyzJTSxJzimpoa"
s: String = eQA5eJwzNDE1sDQ3NDI0tjQ2rjGoMTE0MDI1NjS0BIEa_dSKxNyCnFS9_KJ0fc-8lMyyzJTSxJzimpoa

scala> Zip.decompress(Base64.decodeBase64(s))
res0: String = 1450971213933|0|4102531199999|/example.org/Individuals|||
```